.github/workflows/find-inactive-tsc.yml

```
@@ -41,7 +41,10 @@ jobs:
         with:
           author: Node.js GitHub Bot <github-bot@iojs.org>
           branch: actions/inactive-tsc
-          body: This PR was generated by tools/find-inactive-tsc.yml.
+          body: |
+            This PR was generated by tools/find-inactive-tsc.yml.
+
+            In case there was an error: ${{ env.INACTIVE_TSC_MEMBERS }}
```

```suggestion
            @nodejs/tsc ${{ env.INACTIVE_TSC_MEMBERS }}
```
Wouldn't something like this be enough? (with or without `@nodejs/tsc`)

---

I'm fine with that. @targos is the one who suggested automating some kind of pinging/notification, so...What do you think, @targos?

---

Both LGTM.